src/profiles/abi_music/parsing.py

```diff
@@ -34,10 +48,20 @@ def parse_timestamp(timestamp: str) -> datetime:
     raise ValueError("Ill-formed timestamp; expected format 'yymmdd-DDMMSS'")
 
 
-def parse_project_info(json_data: dict[str, Any]) -> RawProject:
+def read_json(file: FileNode) -> dict[str, Any]:
+    """Extract the JSON data hierachy from `file`"""
+    file_data = file.path().read_text(encoding="utf-8")
+    json_data: dict[str, Any] = json.loads(file_data)
+    return json_data
+
+
+def parse_project_info(directory: DirectoryNode) -> RawProject:
     """
     Extract project metadata from JSON content
     """
+
+    json_data = read_json(directory.file("project.json"))
+
     groups: list[GroupACL] = []
 
```

Comment:
The groups code should be replicated for Users too - I don't know if Greg plans to implement anything with userrs in the future, but it would be good to at least note in the code that here is where you would implement something of the sort